AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -482,38 +482,56 @@ msgstr ""
 "                 %(SE)s: %(data1)s to %(data2)s. With %(total_observed)s "
 "officially notified cases so far "
 
-#: dados/templates/alert_base.html:195
+#: dados/templates/alert_base.html:237
 #, python-format
 msgid " Últimas %(look_back)s semanas: "
 msgstr " Previous %(look_back)s weeks: "
 
-#: dados/templates/alert_base.html:205
+#: dados/templates/alert_base.html:247
 msgid "Suspeita de um criadouro?"
 msgstr "Want to report a breeding site?"
 
-#: dados/templates/alert_base.html:206
+#: dados/templates/alert_base.html:250
+msgid "Em caso de suspeita de criadouro de mosquitos, "
+msgstr ""
+
+#: dados/templates/alert_base.html:253
+msgid "contacte a central de apoio ao cidadão: 1746"
+msgstr "contact the citizen support center: 1746"
+
+#: dados/templates/alert_base.html:256 dados/templates/alert_base.html:265
+#: dados/templates/alert_base.html:268
+msgid "contacte a secretaria de saúde de sua cidade"
+msgstr "contact your local health department"
+
+#: dados/templates/alert_base.html:259
 msgid ""
```

Comment:
if you change moving the translation block into the tag <a> it should be removed automatically. if it is possible always try to avoid using html tags inside translation.

Reply:
ok